datacenter/ucp/2.1/guides/admin/configure/content-trust/admin_tasks.md

```diff
@@ -0,0 +1,387 @@
+you need to also rotate the `snapshots` key, so that private keys do not need
+to be transferred between the client and server.
+
+1.  Rotate the key and configure the Notary server to manage it. This operation
+    only needs to be done once for each trusted repository.
+
+    ```bash
+    $ notary key rotate <GUN> snapshot --server-managed
+    ```
+
+    You are prompted for the DTR credentials followed by the passphrase for the
+    `root` key.
+
+2.  For each user who should be able to sign images, ask that user to create a
+    client bundle. They should:
+
+    1.  Go to the UCP web UI.
+    2.  Click your username at the top right. Click **Profile**.
+    3.  Click **Create a Client Bundle**. A file is downloaded called
+        `ucp-bundle-<username>.zip`.
+    4.  Extract the zip file. The important file within the archive is the
+        `cert.pem`, which is the user certificate.
+    5.  Send you the `cert.pem` **through a secure, trusted channel**. If you
+        plan to create more than one delegation, rename the `cert.pem` with the
+        username or other identifying information.
+
+3.  Run the following command to add the `targets/releases` delegation role for
+    each user, using the `cert.pem` files. You can specify multiple `cert.pem`
+    files at once.
+
+    ```bash
+    $ notary delegation add -p <GUN> targets/releases --all-paths user1.pem user2.pem
+    ```
+
+    You will be prompted for your DTR credentials and the passphrase for the
+    `targets` key.
+
+    > **Note**: You can also add arbitrary delegations, but `targets/releases`
+    > is a special delegation, and is treated as an actual release branch for
+    > Docker Content Trust. If a Docker client has content trust enabled, and
+    > the client runs `docker pull`, this delegation is what signals that the
+    > content is trusted.
+
+    Each user who can release images should be added to the `targets/releases`
+    role.
+
+4.  Create at least one more delegation and add users to it, or UCP will not
+    honor the signed content. This delegation indicates the team that is signing
+    the release.
+
+    Docker recommends adding one delegation per team. For instance, if you have
+    an `engineering` team and a `qa` team, add a delegation for each of these.
+    If a user is a member of both teams, that user will be able to indicate
+    which team they are signing on behalf of. Notary has no limit on how many
+    delegation roles can exist.
+
+    Valid delegation roles take the form of `targets/<delegation>`. Do not include
+    a trailing slash.
+
+    The following command adds `user1` to the `targets/engineering` delegation:
+
+    ```bash
+    $ notary delegation add -p <GUN> targets/engineering --all-paths user1.pem
+    ```
+
+    You will be prompted for your DTR credentials followed by the passphrase
+    for the `targets` key.
+
+5.  Securely remove the `.pem` files of the users you added delegations to. If
```

Comment:
These `.pem` files are certs containing public keys, and can't alone be used to sign - they're certificates that specify the username and the fingerprint of the actual signing key.